CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+1.2.0 (unreleased)
+=====
+
+* (improvement) Add support for Symfony 7.
+* (improvement) Require Symfony 6.4 and PHP 8.2
```

Comment:
The `composer.json` (and the CI.yml require PHP 8.3, though.

Reply:
This was already fixed in the release commit https://github.com/21TORR/TaskManagerBundle/commit/96e3eb7f28d65dfab847277b9c514b3e442f387f